index.html

```diff
@@ -131,7 +131,7 @@
 &lt;/html&gt;
 </pre>
     <p>Alternatively, the developer can observe the <a>Performance Timeline</a>
-    and be notified of new performance metrics via the
+    and be notified of both previously buffered, and new performance metrics via the
```

Comment:
We should clarify that buffering only applies until onload.

"... and be notified of new performance metrics, and optionally buffered performance metrics recorded prior to the `load` event of the document, via the..."